app/models/git/repository.rb

```diff
@@ -13,8 +13,7 @@ def initialize(repo_name, repo_url: nil)
       elsif repo_url
         @repo_url = repo_url
       elsif Rails.env.test?
-        # TODO; Switch when we move back out of monorepo
-        @repo_url = "file://#{Rails.root / 'test' / 'repos' / 'v3-monorepo'}"
+        @repo_url = "file://#{Rails.root / 'test' / 'repos' / 'track-with-exercises'}"
```

Comment:
I'm now defaulting to the `track-with-exercises` repo. We probably want to remove the `v3-monorepo` once this PR is done.